src/content/1.7/development/components/configuration.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Configuration storage
+weight: 80
+---
+
+# Configuration storage
+
+The Configuration storage of PrestaShop aims to address two needs.
+
+First, there is multiple situations where a single value has to be stored and be available easily. It can be a boolean to control order mode, it can be a limit for timeout: a simple key => value(s) storage is needed to manage multiple settings in PrestaShop.
```

Comment:
```suggestion
First, there are multiple situations where a single value has to be stored and be available easily. It can be a boolean to control order mode, it can be a limit for timeout: a simple key => value(s) storage is needed to manage multiple settings in PrestaShop.
```